src/components/ServiceRequestForm.jsx

```diff
@@ -125,6 +125,7 @@ const ServiceRequestForm = (props, errors, touched) => {
 					return requestSubCategory;
 				}
 
+				localProps.setFieldValue('Categories', result.data);
```

Comment:
Is this needed? I think I added this as a test and its not being used for anything. I thought I had deleted it.

Reply:
Yes, this keeps the Categories in the overall form state, so that we can access them.

If you look [here](react-baltcogo_ProvideDetails.jsx at bug-fixes · baltimorecounty_react-baltcogo.html), you will see how we are using it. Unless we change the app's architecture it probably makes sense to store any of the ajax values in the formik props so that we can use them if needed.

The other option would be to do a set field for the Selected Category Object and Selected Subcategory object on the props, but this would also be a big shift.

Each approach didn't seem appropriate so this is the quickest solution, even if it's dirty.

Reply:
Okay, I didnt know we had actually used it. Makes sense but I knew I had added that once before as a test. Hadn't realized that it had become more than just a test now.